pkg/csv/writer.go

```diff
@@ -57,6 +58,20 @@ func (c *GenericWriter) WriteBytes(buf *bytes.Buffer) error {
 	return nil
 }
 
+// WriteBytes writes out csv header and values to the provided buffer
+func (c *GenericWriter) WriteCSV(w io.Writer) error {
```

Comment:
I'm curious, why did you take the approach of adding a new method here instead of following the existing pattern?

Reply:
To write csv data in zip io writer and not create a new buffer like vuln reporting

Reply:
cc: @charmik-redhat to shed light on why it was done differently for vuln. To get better understanding for myself.